.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -191,6 +192,8 @@ public static class Constants
 
         public const string InvalidDateString = "0001-01-01";
 
+        public const char TimexDelimiter = '|';

```

Comment:
Rename to CompositeTimexDelimiter as its not a regular timex delimiter, but only a delimiter when there are multiple timexes in the attribute.